subset/security/.htpasswd

```diff
@@ -0,0 +1 @@
+daquser:$apr1$unLT.q7.$IWSPKV1KMsSaIDh3aq0Ab.
```

Comment:
potentially can these be generate during the docker build ? or the start_faux script ?

Reply:
Password files now generated  in the faux container at runtime instead if being copied over from DAQ directory